apps/studio.giselles.ai/app/workspaces/[workspaceId]/layout.tsx

```diff
@@ -76,7 +76,7 @@ export default async function Layout({
 				runV3,
 				sidemenu,
 				githubTools,
-				webSearchAction,
+				webSearchAction: true,
```

Comment:
```suggestion
				webSearchAction,
```

Reply:
@toyamarinyon Is it possible to switch the flag using the vercel flag?

Reply:
Sorry! I missed. It was for debugging